src/main/webapp/resources/js/pages/user/components/UserDetailsPage.jsx

```diff
@@ -0,0 +1,196 @@
+import React, { useState } from "react";
+import { useParams } from "react-router-dom";
+import {
+  Button,
+  Form,
+  Input,
+  notification,
+  Select,
+  Space,
+  Switch,
+  Typography,
+} from "antd";
+import { formatDate } from "../../../utilities/date-utilities";
+
+import {
+  useGetUserDetailsQuery,
+  useEditUserDetailsMutation,
+} from "../../../apis/users/users";
+
+/**
+ * React component to display the user details page.
+ * @returns {*}
+ * @constructor
+ */
+export default function UserDetailsPage() {
+  const { userId } = useParams();
+  const { data, isSuccess } = useGetUserDetailsQuery(userId);
```

Comment:
Instead of `isSuccess` when getting the user details use `isLoading`

Reply:
Please see 5200c30b6c76bd8bf906ec362711b245693da389.